### PR TITLE
Fix missing files issue with mlflow converted model

### PR DESCRIPTION
**Pull request by novaturient95**
**Problem**
Segregating files based on regex was causing one or more files to miss in many cases, causing failures in local validation. Segregation is still required as FT components does not support reading just from data/model folder

**Solution**
1. Copy all files to model folder
2. Load config and tokenizer

### CODE REVIEW

File: assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py [hunk 1]
```diff
@@ -99,6 +99,8 @@ class GITDownloader:
     """Downloader class for model hosted on public git repositories."""
 
     URI_TYPE = PathType.GIT.value
+    FILES_TO_REMOVE = [".gitattributes"]
+    DIRS_TO_REMOVE = [".git"]
```
Comment (skasturi) [inline assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py:103]: I believe there is an amlignore or something similar as well isn't it?

Comment (skasturi) [reply assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py:103]: .. not specifically in git downloader but potentially when we are uploading/registering the model.

**Review by skasturi — APPROVED** [review]

**Comment by skasturi** [comment]
Please fix the failing gate as well.

File: assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py (hunk 1 above)
Comment (novaturient95) [reply assets/training/model_management/src/azureml/model/mgmt/downloader/downloader.py:103]: added

**Comment by novaturient95** [comment]
somehow loading model config is loading model checkpoints. Checking !!!

image

**Comment by novaturient95** [comment]
> somehow loading model config is loading model checkpoints. Checking !!!
> 
> run: https://ml.azure.com/runs/neat_store_6y0tlh2z8s?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourcegroups/donotuseordelete-model-validation-rg/workspaces/model-validation-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
> 
> image

parameters were incorrectly passed in this case. But need to introduce validation.

**Review by skasturi — APPROVED** [review]

**Comment by novaturient95** [comment]
> > somehow loading model config is loading model checkpoints. Checking !!!
> > run: https://ml.azure.com/runs/neat_store_6y0tlh2z8s?wsid=/subscriptions/ed2cab61-14cc-4fb3-ac23-d72609214cfd/resourcegroups/donotuseordelete-model-validation-rg/workspaces/model-validation-eastus&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
> > image
> 
> parameters were incorrectly passed in this case. But need to introduce validation.

created task: https://dev.azure.com/msdata/Vienna/_workitems/edit/2707653/

**Pull request merged**